configuration/managing-preview-features.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-preview-features"]
+= Managing technology preview features
+include::modules/common-attributes.adoc[]
+:context: managing-preview-features
+
+toc::[]
+
+[role="_abstract"]
+
+You can enable or disable features that are Technology Preview by using feature flags. These features might be incomplete, unstable, and are provided with limited support. Use them at your discretion.
```

Comment:
```suggestion
You can enable or disable features that are in Technology Preview by using feature flags. These features might be incomplete, unstable, and are provided with limited support. Use them at your discretion.
```

Reply:
@kcarmichael08 ?
Also, shouldn't Technology Preview be lowercase, as in the IBM guide?

Reply:
I think it's a special case - we capitalize it in the standard "snippet" we use whenever we refer to Technology Preview features (see https://github.com/openshift/openshift-docs/blob/main/snippets/technology-preview.adoc).

Reply:
(You might want to copy the language from the snippet to use here, since it's standard text that we use for all of the OpenShift docs and is already vetted by legal, compliance, etc.)